core/assets/bundles/bundle_in_ID.properties

```diff
@@ -1236,39 +1236,39 @@ team.green.name = hijau
-hint.command.mobile = [accent][[Double-tap][] your unit to command nearby units into formation.
-hint.payloadPickup = Press [accent][[[] to pick up small blocks or units.
-hint.payloadPickup.mobile = [accent]Tap and hold[] a small block or unit to pick it up.
-hint.payloadDrop = Press [accent]][] to drop a payload.
-hint.payloadDrop.mobile = [accent]Tap and hold[] an empty location to drop a payload there.
-hint.waveFire = [accent]Wave[] turrets with water as ammunition will automatically put out nearby fires.
-hint.generator = \uf879 [accent]Combustion Generators[] burn coal and transmit power to adjacent blocks.\n\nPower transmission range can be extended with \uf87f [accent]Power Nodes[].
+hint.desktopMove = Tekan [accent][[WASD][] untuk bergerak.
+hint.zoom = [accent]Skrol[] untuk membesarkan atau mengecilkan layar.
+hint.mine = Dekati \uf8c4 bijih tembaga dan [accent]tekan[] untuk menambangnya secara manual.
+hint.desktopShoot = [accent][[Klik kanan][] untuk menembak.
+hint.depositItems = Untuk mentransfer item, tekan item dari inti.
+hint.respawn = untuk muncul kembali seperti awal, tekan [accent][[V][].
+hint.respawn.mobile = Kamu telah mengambil alih kontrol dari sebuah unit/struktur. Untuk muncul kembali, [accent]tekan avatar di kiri atas.[]
+hint.desktopPause = Tekan [accent][[Spasi][] untuk menjeda dan menghentikan jeda.
+hint.placeDrill = Pilih \ue85e [accent]Bor[] menu di kanan bawah, lalu pilih \uf870 [accent]Bor[] dan klik diatas bijih tembaga untuk menaruhnya.
+hint.placeDrill.mobile = Pilih \ue85e[accent]Bor[] tab di menu di kanan bawah, lalu pilih \uf870 [accent]Bor[] dan klik diatas bijih tembaga untuk menaruhnya.\n\nTekan \ue800 [accent]checkmark[] di bawah kanan untuk menkonfirmasi pembangunan.
+hint.placeConveyor = Pengantar dapat memindahkan item dari bor ke blok yang lain. Pilih \uf896 [accent]Pengantar[] dari menu \ue814 [accent]Distribusi[].\n\nKlik dan tarik untuk menaruh beberapa pengantar.\n[accent]Skrol[] untuk memutarnya.
+hint.placeConveyor.mobile = Pengantar dapat memindahkan item dari bor ke blok yang lain. Pilih \uf896 [accent]Pengantar[] dari menu \ue814 [accent]Distribusi[].\n\nTahan jari kamu sebentar dan seret untuk menaruh beberapa pengantar.
+hint.placeTurret = Taruh \uf861 [accent]Turret[] untuk mempertahankan basismu daru musuh.\n\nTurret membutuhkan amunisi - yang satu ini, \uf838copper.\nGunakan pengantar dan bor untuk mengisinya.
+hint.breaking = [accent]Klik kanan[] dan tarik untuk menghancurkan blok.
+hint.breaking.mobile = Aktifkan \ue817 [accent]palu[] di kanan bawah dan tekan untuk menghancurkan blok.\n\nTahan jari kamu sebentar dan seret untuk menghancurkannya.
+hint.research = Gunakan tombol \ue875 [accent]Riset[] untuk mempelajari teknologi baru.
+hint.research.mobile = Gunakan tombol \ue875 [accent]Riset[] di \ue88c [accent]Menu[] untuk mempelajari teknologi baru.
+hint.unitControl = Tahan [accent][[L-ctrl][] dan [accent]klik[] untuk mengendalikan unit atau turret teman.
```

Comment:
ambiguous (friendly unit/turret, not friend's unit/turret)

Reply:
friendly entity?

Reply:
pake punya gw klo mau
@grimsoul10 lu maunya gmn?